src/configmanager.cpp

```diff
@@ -106,7 +111,9 @@ bool ConfigManager::load()
 		boolean[BIND_ONLY_GLOBAL_ADDRESS] = getGlobalBoolean(L, "bindOnlyGlobalAddress", false);
 		boolean[OPTIMIZE_DATABASE] = getGlobalBoolean(L, "startupDatabaseOptimization", true);
 
-		string[IP] = getGlobalString(L, "ip", "127.0.0.1");
```

Comment:
is this and port checks really needed?

Reply:
Yes, because when you change IP (or game/login port) from argument it will be set here https://github.com/otland/forgottenserver/pull/2756/files#diff-b7fa32a8694c6de78c4454514026dba8R352 and then during config loading it will overwrite it when detects that value inside config lua file.